sections/semantics-embedded-content.include

```diff
@@ -5867,6 +5867,21 @@ attribute's value is a type that a <a>plugin</a> supports, then the value of the
   The <dfn attribute for="HTMLVideoElement"><code>poster</code></dfn> IDL attribute must
   <a>reflect</a> the <{video/poster}> content attribute.
 
+  <div class="example">
+    This example shows how different video files can be offered to the browser. If the browser does not support a specific codec,
+    it can play one of the alternative files offered.
+
+    This example also shows the <code>controls</code>, and <code>preload</code> attributes.
```

Comment:
The attributes should be linked here, using `<{video/controls}>` and `<{video/preload}>` syntax.